apps/web/src/pages/dao/[token]/vote/[id].tsx

```diff
@@ -127,23 +127,14 @@ export const getServerSideProps: GetServerSideProps = async ({ params, req, res
     }
   }
 
-  if (
-    ethers.utils.getAddress(proposal.collectionAddress) !==
-    ethers.utils.getAddress(collection)
-  ) {
```

Comment:
do we want to remove this? iirc this page would resolve to any proposal no matter the given dao address.

Reply:
good point just added back in